app/controllers/ht_approval_requests_controller.rb

```diff
@@ -3,18 +3,26 @@
 class HTApprovalRequestsController < ApplicationController
   before_action :fetch_requests, only: %i[show update edit]
 
-  def create
-    return unless params[:submit_req]
-
-    adds = add_requests(params[:ht_users])
-    flash[:notice] = "Added #{'request'.pluralize(adds.count)} for #{adds.join ', '}" if adds.count.positive?
+  def create # rubocop:disable Metrics/MethodLength
+    if params[:submit_requests]
+      @added_users = add_requests(params[:ht_users])
+      flash[:notice] = "Added #{'request'.pluralize(@added_users.count)} for #{@added_users.join ', '}" if @added_users.count.positive?
```

Comment:
Consider moving into `add_requests` (and likewise below for the flash & session manipulation for `add_renewals`?)

Reply:
On the list.